vignettes/MSstatsUpdated.Rmd

```diff
@@ -0,0 +1,337 @@
+
+This vignette summarizes the introduction and various options of all functionalities in MSstats. More details are available in `User Manual`.
+
+## __Installation__
+
+To install this package, start R (version “4.0”) and enter:
+
+``` {r code Installation}
+if (!requireNamespace("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+
+BiocManager::install("MSstats")
+library(MSstats)
+
+```
+
+## __1. Workflow__
+
+### __1.1 Raw Data__ 
+
+To begin with, we will load sample datasets, including both annotated and plain data. The dataset you need can be found [here]( https://github.com/Vitek-Lab/MSstatsConvert/blob/devel/inst/tinytest/raw_data/PD/pd_input.csv).
+
+We will also load the Annotation Dataset using MSstatsConvert. You can access this dataset [here](https://github.com/Vitek-Lab/MSstatsConvert/blob/devel/inst/tinytest/raw_data/PD/annot_pd.csv).
+
+
+``` {r code Load Dataset}
+library(MSstats)
+
+# Load data
+pd_raw = system.file("tinytest/raw_data/PD/pd_input.csv", 
+                    package = "MSstatsConvert")
+
+annotation_raw = system.file("tinytest/raw_data/PD/annot_pd.csv", 
+                   package = "MSstatsConvert")
+
+pd = data.table::fread(pd_raw)
+annotation = data.table::fread(annotation_raw)
+
+head(pd, 5)
+head(annotation, 5)
+
+```
+
+
+### __1.2 Converters__
+
+
+
+We have the following converters, which allow you to convert various types of output reports which include the feature level data to the required input format of MSstats. Further information about the converters can be found in the MSStatsConvert package.
+
+1. DIANNtoMSstatsFormat
+2. DIAUmpiretoMSstatsFormat
+3. FragPipetoMSstatsFormat
+4. MaxQtoMSstatsFormat
+5. OpenMStoMSstatsFormat
+6. OpenSWATHtoMSstatsFormat
+7. PDtoMSstatsFormat
+8. ProgenesistoMSstatsFormat
+9. SkylinetoMSstatsFormat
+10. SpectronauttoMSstatsFormat
+11. MetamorpheusToMSstatsFormat
+
+We show an example of how to use the above said Converters. 
+
+```{r Converter Files}
+
+library(MSstatsConvert)
+
+skyline_raw = system.file("tinytest/raw_data/Skyline/skyline_input.csv", 
+                    package = "MSstatsConvert")
+
+skyline = data.table::fread(skyline_raw)
+head(skyline, 5)
+
+```
+
+```{r SkylinetoMSstatsFormat, results='hide', message=FALSE, warning=FALSE}
+
+msstats_format = MSstatsConvert::SkylinetoMSstatsFormat(skyline_raw,
+                                      qvalue_cutoff = 0.01,
+                                      useUniquePeptide = TRUE,
+                                      removeFewMeasurements = TRUE,
+                                      removeOxidationMpeptides = TRUE,
+                                      removeProtein_with1Feature = TRUE)
+
+
+```
+
+```{r SkylinetoMSstatsFormat head}
+head(msstats_format)
+```
+
+
+
+### __1.3 Loading PD Data to MSstats__
```

Comment:
Could we put this section before the skyline converter?  I think for better continuity, we load PD data, then convert it, and then we show another example of a different converter.

Reply:
+ reminder to load MSstatsConvert before you execute this code.